medusa/server/web/config/post_processing.py

```diff
@@ -213,19 +220,13 @@ def isNamingValid(pattern=None, multi=None, abd=False, sports=False, anime_type=
             return 'invalid'
 
     @staticmethod
-    def isRarSupported():
-        """
-        Test Packing Support:
-            - Simulating in memory rar extraction on test.rar file
-        """
-
+    def isRarSupported(unrar_tool):
```

Comment:
Well for starters it may be a bad practice, but it's also a convention we've chosen. One thing I hate more then a bad convention is an implementation that only partially follows a 'bad' convention. So or we do all of them at once, meaning we change the convention for the routes, or we don't.

So if you want to do something about, go ahead. But that does have nothing to do with this PR.